llvm/unittests/DWARFLinkerParallel/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 set(LLVM_LINK_COMPONENTS
+  DWARFLinker
```

Comment:
also seems unrelated

Reply:
This is necessary in general even if everything is working currently without DWARFLinker(as LLVMDWARFLinker contains base functionality).